### PR TITLE
chore: Adds python36 deprecation to changelog

### PR DESCRIPTION
**Pull request by ShubhamChaturvedi7**
*Issue #, if available:*

*Description of changes:*

chore: Adds python36 deprecation to changelog

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+3.2.0 -- 2021-12-19
+===================
+
+Deprecation
+-----------
+The AWS DynamoDB Encryption Client for Python no longer supports Python 3.6
+as of version 3.2; only Python 3.7+ is supported. Customers using
+Python 3.6 can still use the 2.x line of the AWS DynamoDB Encryption Client for Python,
+which will continue to receive security updates, in accordance
+with our `Support Policy <https://github.com/aws/aws-dynamodb-encryption-python/blob/master/SUPPORT_POLICY.rst>`__.
```
Comment (seebees) [inline CHANGELOG.rst:14]: Is this true?
Doesn't  #481 make that not true?

**Review by seebees — APPROVED** [review]
LGTM

File: CHANGELOG.rst (hunk 1 above)
Comment (ShubhamChaturvedi7) [reply CHANGELOG.rst]: Reworded this based on the fact that we are dropping support for 2.x as well.

**Review by seebees — APPROVED** [review]
LGTM

**Pull request merged**